layers/meta-tegrademo/recipes-bsp/u-boot/u-boot-fitimage_1.0.0.bb

```diff
@@ -0,0 +1,18 @@
+DESCRIPTION = "FIT image which have Kernel, DTB and ramdisk"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+COMPATIBLE_MACHINE = "(cot)"
```

Comment:
Any reason why a FIT image requires this restriction?  Can you generalize the FIT image integration to not be restricted to `cot`?

Reply:
FIT image is compatible only when Jetson platform have `u-boot` otherwise this recipe will be not parse by `bitbake`.
That's why I put this restriction.

Reply:
This recipe should used only when build for `Jetson Nano` and `Jetson TX2`